beacon-node-oapi.yaml

```diff
@@ -421,3 +424,13 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/ConsensusVersion'
+    Eth-Execution-Payload-Blinded:
+      description: Required in response so client can deserialize returned json or ssz data to the correct object.
+      required: true
+      schema:
+        type: boolean
+    Eth-Execution-Payload-Value:
```

Comment:
Without the consensus value, there's no (reasonable) way for a VC to compare blocks for profit - should we add this here?

Reply:
Does Nimbus already compute this value during block proposal? Lighthouse sort of does, but approximates slightly (if there are slashable attestations we don't keep track of the rewards from both).

It would make VC block choosing more fair, as the heuristics Vouch uses currently seem to favour Prysm despite Prysm's blocks being less profitable than Lighthouse and Nimbus on average (according to my blockprint nodes, public dash coming soon).

Reply:
could this be a new PR? this one is closed.

Reply:
> Does Nimbus already compute this value during block proposal? 

We do it for offline analysis for historical blocks (to pick out more detail than is needed for state transitions) but making it "online" would probably not be too hard - indeed, the VC currently has to do a lot of guesswork and it will likely always be wrong / skewed - further exacerbating the problem is that in that guesswork, it's not working with eth-denominated data usually but rather "fresh attestation count" and similar proxies which prevents the value from being compared with the execution profit (ie low cl+high el vs low cl+high el score)

Reply:
> could this be a new PR? this one is closed.

https://github.com/ethereum/beacon-APIs/pull/358